tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc

```diff
@@ -60,14 +60,16 @@ https://opentelemetry.io[OpenTelemetry] is a set of APIs, SDKs, tooling and inte
 
 This specification defines the behaviors that allow MicroProfile applications to easily participate in an environment where distributed tracing is enabled via https://opentelemetry.io[OpenTelemetry] (a merger between https://opentracing.io[OpenTracing] and https://opencensus.io[OpenCensus]).
 
-This document and implementations *MUST* comply with the following OpenTelemetry {otel-spec-main-version} specifications:
+The OpenTelemetry specification describes the cross-language requirements and expectations for all OpenTelemetry implementations. This specification is based on the https://github.com/open-telemetry/opentelemetry-java/tree/v{otel-java-version}[Java implementation v{otel-java-version}] of OpenTelemetry. An implementation of this MicroProfile Telemetry can use a later patch release of the Java implemenation.
```

Comment:
I'd propose _binary compatible_ instead of _later patch_. We don't get many patch releases of otel-java, but the stable parts should remain compatible.

Reply:
@pdudits binary comaptible changes include minor updates, which should NOT be allowed. In the above sentence, I tried to clarify only later patch releases are allowed to be consumed not the other releases. This spec should lock down major.minor release but open micro releases.

Reply:
I have reworded a bit.